tools/minimalImageBuildTool/pkg/github/github_test.go

```diff
@@ -0,0 +1,44 @@
+package github_test
```

Comment:
Wondering, do we need the Github pr + issue creation functionality for the minimal images builder? It might be better to just not include the 'github', 'prManager', and 'issueManager' in the tool, since I don't think we'll actually need those for this specific use-case